### ISSUE TITLE
Feature: show company total

### ISSUE THREAD

**Issue by AinizeCidoncha** [microsoft/wpa#22]
**Is your feature request related to a problem? Please describe.**
It is not easy to get the graphs and tables for the whole compnay, without breaking it down into an HR attribure

**Describe the solution you'd like**
One option could be to have the option of typing Hrvar= "Total"

**Describe alternatives you've considered**
It could also be added at the end, or the default when you don't have an HRvar

**Additional context**

**Comment by martinctc** [comment]
Thanks for adding this - this is being implemented in pull request #17! 
The "total" view would be activated if NULL is supplied to the `hrvar` argument, which keeps the function API simple and easy to remember. A new building block function `totals_col()` is added for this purpose. This will become available in the stable version once #17 is approved and merged into the main branch.
